arm-containerregistry/2017-03-01/examples/RegistryDelete.json

```diff
@@ -0,0 +1,12 @@
+{
+    "parameters": {
+        "api-version": "2017-03-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rgname",
+        "registryName": "registry"
+    },
+    "responses": {
+        "200": {},
```

Comment:
Add a 202? Right now we don't clean up storage account. Storage account delete doesn't seem to be async though. https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-12-01/swagger/storage.json#L132